contributing/windows.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Development using Windows
+layout: default
+---
+
+## Setting up the development environment on Windows
+
+This requires Windows 10, 2004 (2020 May Update) or later.
+
+First we need to install the Windows Subsystem for Linux, Visual Studio Code and Docker. Then we will install and configure some software for the WSL2 distribution (Git). There may also be some suggestions on how to make things prettier.
+
+### Visual Studio Code
+
+[VS Code](https://code.visualstudio.com/) is recommended for the ease of integration with WSL2. It should "just work" once the rest of the parts are in place.

```

Comment:
Maybe clarify that VS Code is a text editor? E.g. "VS Code is the recommended text editor because of ease of integration with WSL2".